geotools/src/main/scala/geotrellis/geotools/GridCoverage2DToRaster.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.geotools
+
+import geotrellis.proj4._
+import geotrellis.raster._
+import geotrellis.vector._
+
+import org.geotools.coverage.grid._
+import org.geotools.coverage.grid.io._
+import org.geotools.gce.geotiff._
+import org.osgeo.proj4j._
+
+import scala.collection.JavaConverters._
+import scala.math.{min, max}
+
+
+/**
+  * The [[GridCoverage2DRaster]] object, contains a constructor, as
+  * well as other functions for converting a [[GridCoverage2D]] to a
+  * Geotrellis [[Raster]].
+  */
+object GridCoverage2DToRaster {
+
+  /**
+    * This constructor takes a [[GridCoverage2D]] and returns a
+    * sequence of Geotrellis [[Raster]]s.
+    *
+    * @param  gridCoverage  The  GeoTools GridCoverage2D object
+    * @return               A sequence of rasters of singleband Geotrellis [[Tile]]s
+    */
+  def apply(gridCoverage: GridCoverage2D): Seq[Raster[Tile]] = {
```

Comment:
Is there a reason why this should be `Seq[Raster[Tile]]` and not a `Raster[MultibandTile]`? Is it because the cell types can be different?